NotebookExamples/powershell/Docs/Plotting with Xplot using cmdlets.ipynb

```diff
@@ -187,9 +189,7 @@
    "outputs": [],
    "source": [
     "$count = 20\n",
-    "[datetime[]] $dates = 1..$count | ForEach-Object { (Get-Date).AddMinutes((Get-Random -Minimum $_ -Maximum ($_+30))) }\n",
-    "\n",
-    "$dates.GetType()"
+    "[datetime[]] $dates = 1..$count | ForEach-Object { (Get-Date).AddMinutes((Get-Random -Minimum $_ -Maximum ($_+30))) }"
```

Comment:
You can simplify this:

```suggestion
    "$dates = 1..$count | ForEach-Object { (Get-Date).AddMinutes((Get-Random -Minimum $_ -Maximum ($_+30))) }"
```

Reply:
We shouldn't. That gives back an `Object[]`:

```
IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Object[]                                 System.Array
```

which could be bad when passing something in to a Plotly parameter that has an unbounded type. It's best to be explicit here.